### DISCUSSION TITLE
Can I guide the alignment to be performed by scanorama?

### DISCUSSION THREAD

**Discussion by antonioggsousa** [brianhie/scanorama#123] [Q&A] [answered]
Hi @brianhie,

I was looking into the code of `Scanorama` to know if I could provide some input to guide the alignment/integration of multiple data sets. I know that this performed internally by `Scanorama` probably during [assemble](https://github.com/brianhie/scanorama/blob/master/scanorama/scanorama.py#L97) call which in turn will [find_alignments](https://github.com/brianhie/scanorama/blob/master/scanorama/scanorama.py#L792) if `alignments` and `match` is `None`. 

I was wondering if I could provide directly the order of the alignment in a way similar to what `Seurat` allows in their [IntegrateData()](https://satijalab.org/seurat/reference/integratedata) with the option `sample.tree`. 

I would like to provide the order of integration because I'm trying to integrate more similar and distinct tissues together, although the order that I want it is the most likely order that the software will find and use to align automatically.

Thank you in advance for any advice/recommendation. 
Best regards, 
António

**Comment by brianhie** [comment, marked as answer]
Hi @antonioggsousa,

Apologies for the late reply -- I don't think there's a great way to explicitly specify the order without custom modifications to the code (I think by default the alignment will integrate the smaller dataset into the PCA space of the larger dataset), though by doing the all-to-all comparison I think order will play less of a factor.

**Comment by antonioggsousa** [comment]
Hi @brianhie, 

Thank you for your clarification.

I'll try to see if I can understand how to adapt it to do that.
Although from the results that I'm getting I don't think that I would benefit from giving the order of integration that I intended to give. 

Thank you again for your time. 
Best regards, 
António
